javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,21 @@
+const fullName = 'Jeremy Nguyen';
+const isCool = true;
+const totalPets = 1;
+
+console.log('value of fullName: ' + fullName);
+console.log('value of isCool: ' + isCool);
+console.log('value of totalPets: ' + totalPets);
```

Comment:
You can do console logs this way, but I recommend using `,` instead to separate them. This will make a bit more sense when we get to `javascript-objects`, and I will touch on it in our Q&A